hawc/apps/assessment/api/viewsets.py

```diff
@@ -440,6 +440,40 @@ def logs(self, request: Request, pk: int, type: str):
         return Response(export)
 
 
+class AssessmentAdmin(viewsets.ModelViewSet):
```

Comment:
can this be removed? I think it'd be better if we could change the 
`class Assessment(AssessmentViewSet)` to inherit from the `AssessmentEditViewSet` instead of the current `AssessmentViewSet` class?

And then you could just check on the create/update/delete methods that the user passes the isAdminUser check, otherwise 403.

That'd be much better since we wouldn't have an additional API ViewSet, it would be more DRY. 

Just riffing here (untested):


```python

class Assessment(AssessmentEditViewSet):
    def get_permissions(self):
        if self.action in ['create', 'update', delete']:
            permission_classes = [IsAdminUser]
        else:
            permission_classes = [AssessmentLevelPermissions]
        return [permission() for permission in permission_classes]
```